paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/CvcRecollectionFieldScreenshotTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.stripe.android.paymentsheet.ui
+
+import com.stripe.android.model.CardBrand
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import com.stripe.android.ui.core.elements.CvcConfig
+import com.stripe.android.ui.core.elements.CvcController
+import com.stripe.android.uicore.utils.stateFlowOf
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import org.junit.Rule
+import org.junit.Test
+
+class CvcRecollectionFieldScreenshotTest {
+
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.entries,
+        PaymentSheetAppearance.entries,
+        FontSize.entries
+    )
+
+    @Test
+    fun testEmptyState() {
+        paparazziRule.snapshot {
+            CvcRecollectionField(
+                cvcControllerFlow = cvcControllerFlowWithValue(),
+                isProcessing = false,
+                expanded = true
```

Comment:
I'm not following what these new expanded fields are for. Can this be removed?

Reply:
This is for the paparazzi tests. There is a delay before the animation for the field so the test fails because the view width and height is 0

Reply:
Can we instead supply the animation duration? So it defaults to the static 400ms, but in these tests, we make it 0ms?

Reply:
Updated 👍